collections/bitcoin/_posts/2026-08-28-meta-to-meza.md

```
@@ -0,0 +1,80 @@
+> easily query the DOM with
+> `document.querySelector('[rel="me"][href^="nostr:"]')` or
+> `document.querySelector('[rel="alternate"][href^="nostr:"]')`.
+>
+> <cite>fiatjaf, [NIP PR #1897][pr1897]</cite>
+
+
+
+> I actually prefer "author" on article pages.
+>
+> Because the author might not be me (or the publisher or owner of the
+> website or domain). `rel=me` is defined [here][xfn] for personal web
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)